cpp/data/sql_updates/ub_tools.24

```diff
@@ -0,0 +1,7 @@
+DELETE * FROM metadata_presence_tracer;
+ALTER TABLE metadata_presence_tracer ADD COLUMN marc_subfield_code CHAR(1) NOT NULL AFTER marc_field_tag;
+ALTER TABLE metadata_presence_tracer CHANGE COLUMN zeder_journal_id journal_id INT(11) UNSIGNED;
+ALTER TABLE metadata_presence_tracer DROP CONSTRAINT metadata_presence_tracer_zeder_journal_id;
```

Comment:
The `DROP CONSTRAINT` statements must be executed before the `CHANGE COLUMN` statement

Reply:
... the ADD COLUMN statement could have been stayed at the top because it's semantically not depending on the others, but well... OK anyway